rest/data/project_settings.go

```diff
@@ -219,7 +219,6 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 		}
 		catcher.Wrapf(DeleteSubscriptions(projectId, toDelete), "Database error deleting subscriptions")
 	}
-	fmt.Println("GETTING TO SAVE FOR SECTION")
```

Comment:
did you mean to leave this here?

Reply:
It was definitely an accident that it was here to begin with haha